package.json

```diff
@@ -1,15 +1,29 @@
 {
   "name": "CMR-STAC",
   "version": "2.0.0",
+  "maintainers": [
+    {
+      "name": "Earthdata Search",
+      "email": "support@earthdata.nasa.gov"
+    }
+  ],
   "description": "NASA' Common Metadata Repository STAC implementation.",
   "repository": "https://github.com/NASA/cmr-stac",
   "license": "NASA-1.3",
   "scripts": {
+    "preinstall": "cd cdk/cmr-stac && npm ci --target_arch=x64",
+    "run-synth": "cd cdk/cmr-stac && npm run cdk synth -- --quiet",
+    "run-synth-dev": "cd cdk/cmr-stac-dev && npm run cdk synth -- --quiet",
+    "watch": "onchange 'src/**' -- npm run run-synth-dev",
+    "start:api": "sam local start-api -t ./cdk/cmr-stac/cdk.out/cmr-stac-dev.template.json --warm-containers LAZY --port 3000 --docker-network host",
+    "prestart": "npm run run-synth-dev ",
+    "start": "node start.js",
```

Comment:
This command doesn't work for me. 

```
➜ npm start

> CMR-STAC@2.0.0 prestart
> npm run run-synth-dev


> CMR-STAC@2.0.0 run-synth-dev
> cd cdk/cmr-stac-dev && npm run cdk synth -- --quiet


> cmr-stac@0.1.0 cdk
> cdk synth --quiet

sh: cdk: command not found
```

The preinstall script runs on `cdk/cmr-stac`, not `cdk/cmr-stac-dev`

Reply:
This should be fixed now